### PR TITLE
Remove circular require

### PR DESCRIPTION
**Pull request by johantell**
As they are considered harmful and causes warnings

### CODE REVIEW

**Comment by tobiashm** [comment]
Looks good. Could you rebase or merge master, as the Travis CI build issues should be fixed.

File: lib/openid/fetchers.rb [hunk 1]
```diff
@@ -1,5 +1,4 @@
 require 'net/http'
-require 'openid'
```
Comment (tobiashm) [inline lib/openid/fetchers.rb]: I think you'll need to `require 'openid/version'` instead, as `OpenID::VERSION` is referenced further down in this file.

**Comment by johantell** [comment]
@tobiashm your comments have been adressed.

Thank you for taking a look at this!

**Comment by tobiashm** [comment]
Thanks for the work @johantell

**Pull request merged**